site/docs/demo/showlounge.md

```diff
@@ -15,3 +15,4 @@ Include a description what it does and a link to GitHub—and of course a link t
 - [Deezer History](https://t.me/DeezerHistoryBot) - [Source Code](https://github.com/rojserbest/deezer_history) - Tell others what you were listening to on Deezer.
 - [grammY Documentation Search](https://t.me/grammydocsbot) - [Source Code](https://github.com/grammyjs/website/tree/main/grammydocsbot) - Search and share grammY docs right inside Telegram.
 - [ThemerBot](https://t.me/ThemerBot) - [Source Code](https://github.com/ThemerBot/ThemerBot) - Create themes for most official Telegram apps based on a picture.
+- [Dove Guardarlo?](https://t.me/filmchecker_bot) - [Source Code](https://github.com/Emidio21/FilmChecker-TGBot) - Search which platform (free or flatrate) hosts a movie all around the world.
```

Comment:
Please mark the bot as Italian, too (not just this pull request)

Reply:
Sorry added